### DISCUSSION TITLE
Medic/Genetics thoughts

### DISCUSSION THREAD

**Discussion by Exa644** [FakeFishGames/Barotrauma#7727] [Ideas and suggestions]
So, after playing medic for a bit, a couple things came to mind;
The basic gist is that I think there's too few "medic related" tasks to do on the fly, but also a desire to see some of the current systems more grounded;

-Genetics system:
It is a kinda neat idea, but I overall find it very "fire and forget". You do it as more of an afterthought between missions.

I think it could be interesting to adding some general stat like "alien understanding" or something, that you can improve in various ways. The direct result would be that genetics would be applied with a modifying % or similar (Understanding it well = better; Or even have some options locked behind it)

The basic gist would be two-fold:
--Give poisons more worth in pve; the more "Intact" the creature, the better your result
--Give more reason to have medics EVA (To obtain this stat)
The stat could be gained from scanning enemies in several fashions
--Some handheld scanner. Alive creatures might yield even more (Since large risk to yourself)
--Some ship-mounted scanner (Similar to a weapon), but expensive. Could have secondary scan-uses optionally.
--Some sort of machine that can scan analyze/extract from bodies that have boarded (Also an incentive to do something with those rather than just having them lay about)
While it would make more sense to have per-monster-type scans, an overall stat seems more gameplay-friendly (and more engaging really)

A second part of this could be "after administrations", where you can apply another modifier if you inspect your crew every so often or SOMETHING (I havn't quite cooked this idea), but some sort of "check up on the mutations every so often" as you likely would in an actual scenario

-Growing
Mostly by accident,  but during our play, the medic got turned into a botanist as a side-gig. While this is still hardly occupying since it's quite low maintenance, it does seem like a logical activity.. One that could be encouraged by growing stuff that can make opiums or bandages.
Both would likely have to be a tad more expensive to not make this option too good, but it seems like a realistic thing to do, and somewhat a fun side-thing

-Some specific thing against GSWs and Lacerations that's not just morphine
There's a lot of medical options. But most seem to boil down to "bandage bleedings, apply morphine", for "everyday cases". 
Lacerations and GSWs especially feel quite silly to treat with morphine, so having something dedicated to it wouldn't be amiss imo

-Synergies with missions, like terrorists, where you get better rewards for not killing (higher risk, higher reward deal)
Just some more incentive to have a medic about, and doing its thing. Would have to be in a non-tedious manner tho, since noone really would find it too fun to constantly subdue someone for a minor gain I'd think. But would add more value to poisons here aswell (And a dilemma for wether to treat a prisoner or not etcetc)

-Fix medical UI to always list "what is recommended", and to have some default text like "No bodypart selected" would be helpful
Fairly self-explanatory, but I've found that some stuff lacks "how to treat" suggestions

On a sidenote to this, I've found it mildly unintuitive not being able to double-click the treatment option in the UI, but that's not a major problem

I'd also quite like for recipies to have a hover-tooltip denoting what is needed to craft. Trying to read what is greyed out, and having to click etc is on the tedious side

**Comment by Exa644** [comment]
On an addition to this;
I've been made aware of quality of items not really applying to consumable. I can't say I've extensively tested this, but assuming that is the case, it does leave less value to higher medic skill compared to the other branches?
Again, assuming my understanding is correctly, the applications of medicine just need a skill-threshold to be met, but there's no real benefit of being "better" than that?

**Comment by Exa644** [comment]
And this bit might more apply to the talent-tree as a whole, rather than specifically medic; But I would rather like there to be hover-tooltips for the unlocks like "Unlocks <pressure stabilizer>" or "gives you <book of >"
Without having to look these up on the wiki, or already being familiar with them, it makes the choice of talents somewhat harder

**Comment by Exa644** [comment]
Oh, one more note. The HUD item that gives descriptions;
It lists genetic stuff as orange, the same color as stuff like burns etc. This isn't particularly helpful - be ideal to have those non-ilnesses listed in another color
(I've a similar gripe with buff being *red* in that, like clown power)

**Comment by Exa644** [comment]
One more note on this is the effects in the heal screen; They often swap positions (While you are looking at it), which is less than ideal

**Comment by gerakis** [comment]
Medics need a surgery minigame. Something like surgeon simulator would be incredible. Captain slams into an ice wall and you accidentally sever his artery. That would be amazing. It doesnt even have to be 3d and complex, just a simple "operation" style game would be a lot of fun.

Surgical options would include removing turmors from too many rads, removing parasites, installing implants, clamping veins to stop bleeding, organ swaps from dead donors to healthy patients, etc. You could even have organ harvesting to sell for money at stations.
